Dockerfile

```diff
@@ -70,16 +71,8 @@ RUN chmod +x /download_models.sh && /download_models.sh && rm /download_models.s
 # Install dependencies
 ADD dev-requirements.txt /dev-requirements.txt
 ADD requirements.txt /requirements.txt
-# Manually ignore rocm dependencies
-RUN cget -p $PREFIX ignore \
-    RadeonOpenCompute/clang-ocl \
-    ROCm-Developer-Tools/HIP \
-    ROCmSoftwarePlatform/MIOpen \
-    ROCmSoftwarePlatform/MIOpenGEMM \
-    ROCmSoftwarePlatform/rocBLAS
-RUN cget -p $PREFIX init --cxx /opt/rocm/llvm/bin/clang++
-RUN cget -p $PREFIX install -f dev-requirements.txt
-RUN cget -p $PREFIX install oneapi-src/oneDNN@v1.7
+COPY ./tools/install_prereqs.sh /
+RUN chmod +x /install_prereqs.sh && /install_prereqs.sh /usr/local / && rm /install_prereqs.sh
```

Comment:
Is the `chmod` command needed? Doesn't `COPY` copy the file permissions?

And should this do `/install_prereqs.sh $PREFIX` so the dependencies are installed in `$PREFIX`?

Reply:
removed the chmod line.
the command line is actually be "/install_prereqs.sh /usr/local /". I put two arguments here because the command 
```
cget -p $PREFIX install -f ${REQ_FILE_DIR}dev-requirements.txt
```
requires the `dev-requirement.txt` at `/dev-requiement.txt`.